src/constants.tsx

```diff
@@ -19,6 +19,14 @@ export function isValidNetworkName(value: string): value is NetworkName {
   return value in networks;
 }
 
+export const defaultVerificationServiceUrl =
+  "https://verify.welldonestudio.io/aptos";
+
```

Comment:
If there's a service for this, we should possibly provide help details somewhere for how someone can run it themselves.  Since, this is essentially trusting this URL entirely, it would be good to provide a more trustless system (you can run it yourself), or that the foundation will run it.

Reply:
I will inform you once I have prepared for it.